### DISCUSSION TITLE
DNS & Configuration

### DISCUSSION THREAD

**Discussion by Ethical-Saleem** [vercel/community#6857] [Help] [answered]
### Summary

I deployed my site earlier on vercel. Added a custom domain, cargocreed.com which redirects to www.cargocreed.com. When i visited the domain, it worked. But now the browser is showing "This site cannot be reached". But when i visit the site using vercel's own domain https://cargo-creed.vercel.app/, the site is working well. I have done all the troubleshooting steps I know, but still the same issue
<img width="960" alt="site-error" src="https://github.com/vercel/community/assets/106458354/6f04c770-f768-4e0c-bb30-035e3a0dd6f4">
<img width="960" alt="site-good" src="https://github.com/vercel/community/assets/106458354/f6f30fdf-10dc-4a2d-aca9-7b4691703b5f">

### Steps to Reproduce

<img width="960" alt="site-error" src="https://github.com/vercel/community/assets/106458354/8dafd173-97ed-4391-8cd4-1ded54aa286c">
<img width="960" alt="site-good" src="https://github.com/vercel/community/assets/106458354/1c0dbcac-5e97-4c02-aff9-c45bf9ae8155">

**Comment by swarnava** [comment, marked as answer]
Some of ISP has blocked Vercel in Nigeria unfortunately. You can follow this thread to learn more: https://github.com/orgs/vercel/discussions/3630
